### PR TITLE
 Remove Duplicate Require Statements and Unify Syntax on '

### PR DESCRIPTION
**Pull request by casesandberg**
Duplicate Imports:

https://github.com/casesandberg/atom/commit/8c5c795b35499996b057041c7cabd05f047fe9ae#diff-f93881c6debdf6136cc7fbd22c21340dL1

https://github.com/casesandberg/atom/commit/8c5c795b35499996b057041c7cabd05f047fe9ae#diff-f42f793b4715cfbf8d07c772bc142849L10

### CODE REVIEW

File: spec/babel-spec.coffee [hunk 1]
```diff
@@ -3,9 +3,9 @@
 # See https://github.com/atom/electron/issues/2033
 process.env.DEBUG='*'
 
-path = require('path')
-temp = require('temp').track()
-CompileCache = require('../src/compile-cache')
+path = require 'path'
+temp = require('temp').track();
```
Comment (50Wliu) [inline spec/babel-spec.coffee]: :fire: `;`

File: spec/compile-cache-spec.coffee [hunk 2]
```diff
@@ -1,5 +1,5 @@
 path = require 'path'
-temp = require('temp').track()
+temp = require('temp').track();
```
Comment (50Wliu) [inline spec/compile-cache-spec.coffee]: Same thing; semicolon not needed

**Comment by casesandberg** [comment]
My bad 👻

**Comment by casesandberg** [comment]
@50Wliu fixed

**Comment by 50Wliu** [comment]
Thanks!

**Pull request merged**